src/evict/evict_lru.c

```diff
@@ -1617,6 +1620,22 @@ __evict_walk(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue)
         goto retry;
     }
 
+#ifdef HAVE_DIAGNOSTIC
+    if (!F_ISSET(conn, WT_CONN_IN_MEMORY) && cache->evict_aggressive_score >= WT_EVICT_SCORE_MAX &&
```

Comment:
Nit: `&& cache->evict_aggressive_score == WT_EVICT_SCORE_MAX`

Reply:
@etienneptl suggested to change this to >=

Reply:
My view on this:

- `==` is a strict equality which assumes the max we can have is actually  `WT_EVICT_SCORE_MAX`
- What if the code is broken somewhere else and the score goes beyond `WT_EVICT_SCORE_MAX`? We lose the behaviour we are trying to implement here.
- Unless we are very sure that we are always within bounds, we can keep the `==` but who knows if that guarantee will stay

Reply:
@Siddhartha8899 made the good point on Slack that it can create confusion and make the reader think it can go beyond `WT_EVICT_SCORE_MAX`, which is false.
I am happy to go back to `==` to set the expectations right.

Reply:
ok